plugins/airbrake/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@backstage/plugin-airbrake",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test --coverage",
```

Comment:
Test script should be just `backstage-cli test`, `--coverage` flag can be added when needed

Reply:
Yup, I just saw CI adding it in. Removed!